src/lib/ngsi/Scope.cpp

```diff
@@ -97,16 +97,24 @@ int Scope::fill
 )
 {
   Geometry                    geometry;
-  Georel                      georel;
   std::vector<std::string>    pointStringV;
   int                         points;
   std::vector<orion::Point*>  pointV;
 
+  if (apiVersion == "v1")
+  {
+    type = FIWARE_LOCATION;
+  }
+  else
+  {
+    type = FIWARE_LOCATION_V2;
+  }
+
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/92aaa61d837588d35039bc4af28a98af1cdfbba9